RELEASING.md

```diff
@@ -18,8 +18,7 @@ by 1 and `z` is reset to 0 on the next release cycle.
 
 ```shell
 $ export GITHUB_TOKEN=abcdef1234 # Need Repos scope for update release notes
-$ export SKYGEAR_VERSION="0.5.0"
-$ export KEY_ID="12CDA17C"
+$ export SKYGEAR_VERSION="2.1.0"
```

Comment:
In this file, the first section should also be irrelevant now. We may want to remove it too.